app/controllers/registrations_controller.rb

```diff
@@ -269,6 +269,9 @@ def do_add
 
   def register
     @competition = competition_from_params
+    if current_user
+      @is_processing = Rails.cache.read(CacheAccess.registration_processing_cache_key(@competition.id, current_user.id)) || false
+    end
```

Comment:
```suggestion
    @is_processing = current_user.present? && Rails.cache.read(CacheAccess.registration_processing_cache_key(@competition.id, current_user.id))
```

Reply:
does nil evaluate to false when passed to react?

Reply:
How would my suggested version ever evaluate to nil?

Reply:
oh I guess when Rails.cache.read returns nil then false is returned?

Reply:
Yup `true && nil` is false because `nil` acts as false-y in this context

Reply:
well.. actually `true && nil` is `true`